.ci.yaml

```diff
@@ -3138,6 +3138,21 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac flavors_test_macos
+    bringup: true
+    presubmit: false
```

Comment:
> I verified that this works locally, but I don't know if it's possible to run this on the devicelab try pool to verify that it works on devicelab hardware.

If you make this `true` will it run in the devicelab in this PR?  And then revert it back to `false` before you merge?

Reply:
I don't think this is possible. Setting `presubmit` to `true` didn't make it run, and the ci.yaml analysis check will fail if `bringup` is not set to `true`. However, I was able to manually run a task to get this tested: https://ci.chromium.org/raw/build/logs.chromium.org/flutter/led/andrewkolos_google.com/ec64b67b7abc35f5557ddd021c295affa0b063a77b488f2d920061e99900a674/+/build.proto?server=chromium-swarm.appspot.com